### PR TITLE
add global rocks opts

### PR DESCRIPTION
**Pull request by evanmcc**

### CODE REVIEW

File: src/relcast.erl [hunk 1]
```diff
@@ -190,7 +190,10 @@ start(ActorID, ActorIDs, Module, Arguments, RelcastOptions) ->
     OpenOpts2 = proplists:get_value(db_opts, RelcastOptions, []),
     WriteOpts = proplists:get_value(write_opts, RelcastOptions, [{sync, true}]),
     OpenOpts = OpenOpts1 ++ OpenOpts2,
-    DBOptions = DBOptions0 ++ OpenOpts,
+
+    GlobalOpts = application:get_env(rocksdb, global_opts, []),
+    DBOptions = DBOptions0 ++ OpenOpts ++ GlobalOpts ++
+        [{block_based_table_options, [{no_block_cache, true}]}],
```
Comment (Vagabond) [inline src/relcast.erl]: Are these options still important?

Comment (evanmcc) [reply src/relcast.erl]: no, we could likely remove them.

Comment (Vagabond) [reply src/relcast.erl]: So should we?

Comment (evanmcc) [reply src/relcast.erl]: done.

**Pull request merged**